frontend/src/views/PublicCanteenStatisticsPage/index.vue

```diff
@@ -65,8 +65,7 @@
                   : "s ont publié leurs données (répertoriées dans"
               }}
               <!-- eslint-disable-next-line prettier/prettier-->
-              <router-link :to="{ name: 'CanteensHome' }">nos cantines</router-link>
-              ).
+              <router-link :to="{ name: 'CanteensHome' }">nos cantines</router-link>).
```

Comment:
Marrant, on dirait qu'on a une différence de linter... le mien ajoute un retour à la ligne, le tien l'enlève. As-tu fait un `npm install` récemment ? Je vais le faire de mon coté.

Reply:
I temporarily disable 'format on save' for this file so that we don't end up with a space between the text and the bracket, which looks weird. I'm going to add a backlog task to see if we can change some settings so that we don't end up with spaces in between links and full stops/brackets.